core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/buffer_v1_test.go

```diff
@@ -50,6 +51,96 @@ func TestLogEventBufferV1_SyncFilters(t *testing.T) {
 	require.Equal(t, 1, buf.NumOfUpkeeps())
 }
 
+type readableLogger struct {
```

Comment:
Why do we need this?

Reply:
Just for the purposes of catching and validating log messages within the tests